reports/dasharo-hcl-report

```diff
@@ -239,6 +239,11 @@ cbmem > logs/cbmem.log 2> logs/cbmem.err.log
 update_result "CBMEM table information" logs/cbmem.err.log
 printf '############################################                       |\r'
 
+# echo "Getting CBMEM console..."
+cbmem -1 > logs/cbmem_console.log 2> logs/cbmem_console.err.log
+update_result "CBMEM console" logs/cbmem_console.err.log
+printf '############################################                       |\r'
```

Comment:
Those progress bars need shifting by one `#` to keep consistency

Reply:
from what I see, they actually shift by 2 in each step. Fixed https://github.com/Dasharo/dts-scripts/compare/c77efcbfeaec17228be2da9ab1c1143bcfd42162..05b7c26ef3a6329d2923ba32084c07f501abcf8e